src/policy/immix/block.rs

```diff
@@ -68,6 +68,13 @@ impl BlockState {
 #[derive(Debug, Clone, Copy, PartialOrd, PartialEq)]
 pub struct Block(Address);
 
+impl Default for Block {
```

Comment:
A block starting at address zero is not really a valid block. I am not comfortable with having an invalid default.

Reply:
I need a default block value to initialize a pre-allocated array as the thread-local block allocation/release buffer. I'll see if I can refactor this code...

Reply:
fixed